use business_logic::exceptions::Exception;
use storage::Storage;


/*struct StorageMock {}

impl Storage for StorageMock {
    
    fn get_private_key(&self) -> Result<Option<String>, Exception> {
        return Ok(None)
    }
}

#[test]
fn check_created_account() {
    
}

#[test]
fn check_no_account() {
    
}

#[test]
fn check_account_fail() {
    
}*/